core/gdxsv/gdxsv.cpp

```diff
@@ -142,6 +142,7 @@ std::string Gdxsv::GeneratePlatformInfoString() {
     ss << "maxlag=" << (int) maxlag << "\n";
     ss << "patch_id=" << symbols[":patch_id"] << "\n";
     ss << "machine_id=" << os_GetMachineID() << "\n";
+    ss << "wireless=" << (os_GetConnectionMedium() == "Wireless") << "\n";
```

Comment:
I want it to be an integer explicitly, so please add `int` cast.

Reply:
done